test_conformance/gl/test_images_getinfo_common.cpp

```diff
@@ -97,10 +96,6 @@ static int test_image_format_get_info(cl_context context,
     if (fmt->type == kHalf)
     {
         if (DetectFloatToHalfRoundingMode(queue)) return 0;
```

Comment:
This isn't something changed in this PR, but we're also calling `DetectFloatToHalfRoundingMode` again a few lines below.  Here, we return 0 (success) if the detection fails.  Below, we return 1 (failure) if the detection fails.

Do we need both calls?  (I don't think we do.)
Should we return success or failure if the detection fails?  (Probably failure?)

It's fine to fix this in a separate PR.